components/admin/AdminLayout.tsx

```diff
@@ -27,9 +28,10 @@ export const AdminLayout: React.FC<GetAppProps & { title?: string }> = ({
 
   if (!isAdmin) {
     return (
-      <Layout title={title}>
-        <h1>You must be admin to access this page</h1>
-      </Layout>
+      <ErrorPage
+        code={403}
```

Comment:
Please use the `StatusCode` enum here instead of hardcoding the value for the `code` prop